README.md

```diff
@@ -0,0 +1,2 @@
+# beaker-next-design
+Design and use cases for the new beaker UI
```

Comment:
Can we make this more descriptive? It is the first thing that a contributor can see when you open repository.